.github/workflow_scripts/build_all_docs.sh

```diff
@@ -53,28 +53,25 @@ else
     S3_PATH=s3://$BUCKET/build_docs/$BRANCH/$COMMIT_SHA/all  # We still write to BRANCH so copy_docs.sh knows where to find it
 fi
 
-mkdir -p docs/_build/rst/tutorials/
-aws s3 cp $BUILD_DOCS_PATH docs/_build/rst/tutorials/ --recursive
-
 setup_build_contrib_env
 install_all_no_tests
 
-sed -i -e "s@###_PLACEHOLDER_WEB_CONTENT_ROOT_###@http://$site@g" docs/config.ini
-sed -i -e "s@###_OTHER_VERSIONS_DOCUMENTATION_LABEL_###@$other_doc_version_text@g" docs/config.ini
-sed -i -e "s@###_OTHER_VERSIONS_DOCUMENTATION_BRANCH_###@$other_doc_version_branch@g" docs/config.ini
```

Comment:
With the new website how do we generate link to stable/dev one dynamically?

Reply:
These ad-hoc replacement strings were used in the nav-bar of the old website to link to things like versions, api, tutorials, etc. These exact links don't exist on the new webpage, but their equivalent sections are handled by the sphinx build process automatically. I believe these only needed to be done this way because the links were hard coded in the config file, not built by sphinx.